rs/embedders/src/wasmtime_embedder/linker.rs

```diff
@@ -642,16 +642,16 @@ pub fn syscalls<
     linker
         .func_wrap("ic0", "env_var_value_copy", {
             move |mut caller: Caller<'_, StoreData>,
+                  name_src: I,
+                  name_size: I,
```

Comment:
The fact that CI was green before suggests we're lacking test coverage, e.g., in this module: rs/embedders/tests/wasmtime_embedder.rs.